File: auto-docs/executables/r/show_legend.r
# Learn about API authentication here: plot.ly/r/getting-started
# Find your api_key here: plot.ly/settings/api

library(plotly)
py <- plotly(username='TestBot', key='r1neazxo9w')


trace1 <- list(
  x = c(0, 1, 2), 
  y = c(1, 2, 3), 
  name = "First Trace", 
  showlegend = FALSE, 
  type = "scatter"
)
trace2 <- list(
  x = c(0, 1, 2, 3), 
  y = c(8, 4, 2, 0), 
  name = "Second Trace", 
  showlegend = TRUE, 
  type = "scatter"
)
data <- list(trace1, trace2)
response <- py$plotly(data, kwargs=list(filename="show-legend", fileopt="overwrite"))
url <- response$url

File: auto-docs/executables/julia/show_legend.jl
# Learn about API authentication here: plot.ly/julia/getting-started
# Find your api_key here: plot.ly/settings/api

using Plotly
Plotly.signin("TestBot", "r1neazxo9w")


trace1 = [
  "x" => [0, 1, 2], 
  "y" => [1, 2, 3], 
  "name" => "First Trace", 
  "showlegend" => false, 
  "type" => "scatter"
]
trace2 = [
  "x" => [0, 1, 2, 3], 
  "y" => [8, 4, 2, 0], 
  "name" => "Second Trace", 
  "showlegend" => true, 
  "type" => "scatter"
]
data = [trace1, trace2]
response = Plotly.plot(data, ["filename" => "show-legend", "fileopt" => "overwrite"])
plot_url = response["url"]
